utils/rpms/packaging/rpm_chrootbuild

```diff
@@ -44,4 +44,4 @@ else
     echo "Unable to update /etc/mock/$CHROOT_NAME.cfg."
     echo "You need to make sure it has the needed repos in it yourself."
 fi
-eval mock -r "$CHROOT_NAME" $MOCK_OPTIONS $RPM_BUILD_OPTIONS "$TARGET"
+eval mock --enable-network -r "$CHROOT_NAME" $MOCK_OPTIONS $RPM_BUILD_OPTIONS "$TARGET"
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]